action.yml

```diff
@@ -114,28 +124,57 @@ runs:
       shell: bash
       env:
         ANTHROPIC_API_KEY: ${{ inputs.anthropic-api-key }}
+        OPENAI_API_KEY: ${{ inputs.openai-api-key }}
         GITHUB_TOKEN: ${{ inputs.github-token }}
         KODELET_LOG_LEVEL: ${{ inputs.log-level }}
       run: |
         # Set timeout for command execution
         timeout_minutes="${{ inputs.timeout-minutes }}"
         timeout_seconds=$((timeout_minutes * 60))
+        
+        # Parse and export additional environment variables
+        additional_env='${{ inputs.env }}'
+        if [ "$additional_env" != "{}" ] && [ -n "$additional_env" ]; then
+          echo "Setting additional environment variables..."
+          # Use jq to parse JSON and export each key-value pair
+          echo "$additional_env" | jq -r 'to_entries[] | "export \(.key)=\(.value | @sh)"' > /tmp/env_vars.sh
+          source /tmp/env_vars.sh
+          rm -f /tmp/env_vars.sh
+        fi
+
+        # Handle Kodelet configuration file
+        config_file=""
+        if [ -n "${{ inputs.kodelet-config }}" ]; then
+          # Use specified config file
+          config_file="${{ inputs.kodelet-config }}"
+          echo "Using specified Kodelet config: $config_file"
+        elif [ -f "./kodelet-config.yaml" ]; then
+          # Use default config file if it exists
+          config_file="./kodelet-config.yaml"
+          echo "Using default Kodelet config: $config_file"
+        fi
+
+        # Build config flag if config file is specified
+        config_flag=""
+        if [ -n "$config_file" ]; then
+          config_flag="--config $config_file"
```

Comment:
`kodelet` does not have a `--config` flag, it will auto detect `kodelet-config.yaml` in the current dir, or `~/.kodelet/config.yaml`